test/conformance/testing/uur/fixtures.h

```diff
@@ -312,6 +337,10 @@ struct urMemBufferQueueTest : urQueueTest {
 struct urUSMDeviceAllocTest : urQueueTest {
     void SetUp() override {
         UUR_RETURN_ON_FATAL_FAILURE(uur::urQueueTest::SetUp());
+        if (!GetDeviceInfo<bool>(this->device,
+                                 UR_DEVICE_INFO_USM_DEVICE_SUPPORT)) {
+            GTEST_SKIP() << "Device USM in not supported";
+        }
```

Comment:
I think the checks on the other [PR](https://github.com/oneapi-src/unified-runtime/pull/305) was more safe. so, I suggest we make it similar.

Reply:
Done.